Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -168,6 +172,20 @@ public final class Chat {
         try await channelUpdater.removeMembers(currentUserId: currentUserId, cid: cid, userIds: Set(members), message: systemMessage)
     }
     
+    // MARK: Member State Observing
+    
+    /// Returns an observable member state for the specified user id.
+    ///
+    /// The member state is refreshed before returning the observable state.
+    ///
+    /// - Parameter userId: The user id of the channel member.
+    ///
+    /// - Returns: An instance of `MemberState` which conforms to the `ObservableObject`.
+    public func makeMemberState(for userId: UserId) async throws -> MemberState {
```

Comment:
how would this work? You need to explicitly call this to listen to a specific member? I would expect to listen to a list of members and their updates in the chat state.

Reply:
That is a good point, it is kind of pointless. I added it since we had MemberController. Let me get rid of it. I have another PR for member list.